server/formatter.js

```diff
@@ -4,6 +4,8 @@ const qs = require('querystring')
 const unescape = require('unescape')
 const list = require('./list')
 
+/* Your one stop shop for all your document processing needs. */
+
 const allowInlineCode = (process.env.ALLOW_INLINE_CODE || '').toLowerCase() === 'true'
```

Comment:
It would be nice to get #66 merged first as it could cause some merge conflicts here